INSTALL.md

```diff
@@ -106,9 +106,11 @@ sudo apt-get install libtool autotools-dev automake checkinstall check libopus-d
 
 Fedora:
 ```bash
-yum install libtool autoconf automake check check-devel
+yum install libtool autoconf automake check check-devel  ffmpeg-devel 
```

Comment:
Actually, sorry, it should be under `GCC, Qt, FFmpeg, OpanAL Soft and qrencode` section, not `Clone qTox`